automotive/maliput/utility/generate_obj.cc

```diff
@@ -132,6 +133,36 @@ class GeoNormal {
   api::GeoPosition n_;
 };
 
+// A line segment between two vertices
+class GeoLineSegment {
+  static constexpr double kDistanceTolerance{1e-6};
+
+ public:
+  GeoLineSegment() = delete;
+
+  GeoLineSegment(const GeoVertex& start, const GeoVertex& end)
+      : start_(start), end_(end) {}
+
+  bool Includes(const GeoVertex& vertex) {
+    return Includes(vertex, kDistanceTolerance);
+  }
+
+  bool Includes(const GeoVertex& vertex, double tolerance) {
+    const auto start_pos = start_.v().xyz();
```

Comment:
Sorry, I didn't put it before for all the cases but it applies. Consider replacing `auto` by `const Vector3<double>`

Reply:
Done